src/main/src/main/java/com/shevtsov/dao/impl/ClientDBDao.java

```diff
@@ -0,0 +1,155 @@
+        return -1;
+    }
+
+    @Override
+    public Client findByID(long id) {
+        try (Connection connection = DriverManager.getConnection(DBCostants.DB_URL, DBCostants.LOGIN,
+                DBCostants.PASSWORD); Statement statement = connection.createStatement();) {
+            try (ResultSet resultSet = statement.executeQuery("SELECT * FROM CLIENT WHERE ID = '" + id + "'");) {
+                if (resultSet.next()){
+                    long clientID = resultSet.getLong(1);
+                    String name = resultSet.getString(2);
+                    String surname = resultSet.getString(3);
+                    int age = resultSet.getInt(4);
+                    String phone = resultSet.getString(5);
+                    String email = resultSet.getString(6);
+                    return new Client(clientID, name, surname, age, phone, email);
+                }
+            }
+        } catch (SQLException e) {
+            System.out.println("SOMETHING GOING WRONG!!!");
+        }
+        return null;
+    }
+
+    @Override
+    public void remove(long id) {
+        try (Connection connection = DriverManager.getConnection(DBCostants.DB_URL, DBCostants.LOGIN,
+                DBCostants.PASSWORD); PreparedStatement statement = connection.prepareStatement(
+                        "DELETE FROM CLIENT WHERE ID = ?")) {
+            statement.setLong(1, id);
+            statement.execute();
+        } catch (SQLException e) {
+            System.out.println("SOMETHING GOING WRONG!!!");
+        }
+    }
+
+    @Override
+    public List<Client> gatAll() {
+        try (Connection connection = DriverManager.getConnection(DBCostants.DB_URL, DBCostants.LOGIN,
+                DBCostants.PASSWORD); Statement statement = connection.createStatement()) {
+            try (ResultSet resultSet = statement.executeQuery("SELECT * FROM CLIENT")) {
+                List<Client> clients = new ArrayList<>();
+                while (resultSet.next()) {
+                    long clientID = resultSet.getLong(1);
+                    String name = resultSet.getString(2);
+                    String surname = resultSet.getString(3);
+                    int age = resultSet.getInt(4);
+                    String phone = resultSet.getString(5);
+                    String email = resultSet.getString(6);
+                    clients.add(new Client(clientID, name, surname, age, phone, email));
+                }
+                return clients;
+            }
+        } catch (SQLException e) {
+            System.out.println("SOMETHING GOING WRONG!!!");
+        }
+        return null;
+    }
+
+    @Override
+    public long findByPhone(String phone) {
+        try (Connection connection = DriverManager.getConnection(DBCostants.DB_URL, DBCostants.LOGIN,
+                DBCostants.PASSWORD); Statement statement = connection.createStatement()) {
+            try (ResultSet resultSet = statement.executeQuery("select id from client where phone = '" +
```

Comment:
You can combine this two try blocks

Reply:
like there, 1st answer? (https://stackoverflow.com/questions/8066501/how-should-i-use-try-with-resources-with-jdbc)